openshift/cronJob-template.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: graph-backup-cronjob
+  annotations:
+    description: >
+      This is Thoth Graph Backup CronJob CronJob, this template is meant to be used by Bots,
+      but could also be used by humans...
+    openshift.io/display-name: "Thoth: Graph Backup Job"
+    version: 0.1.0
+    tags: thoth,graph-backup,ai-stacks,aistacks,graph
+    template.openshift.io/documentation-url: https://github.com/Thoth-Station/
+    template.openshift.io/long-description: >
+      This template defines resources needed to deploy Thoth Graph Backup Job CronJob to OpenShift.
+    template.openshift.io/provider-display-name: "Red Hat, Inc."
+    thoth-station.ninja/template-version: 0.1.0
+  labels:
+    template: graph-backup-cronjob
+    app: thoth
+    component: graph-backup
+
+parameters:
+  - description: Registry the ImageStream to be use lives in
+    displayName: ImageStream Registry
+    required: true
+    name: IMAGE_STREAM_REGISTRY
+    value: "docker-registry.default.svc:5000"
+
+  - displayName: Suspend CronJob run
+    description: Suspend CronJob run
+    required: true
+    name: THOTH_SUSPEND_JOB
+    value: "true"
+
+  - description: Project the ImageStream to be use lives in
+    displayName: ImageStream Project
+    required: true
+    name: IMAGE_STREAM_NAMESPACE
+    value: "thoth-infra-stage"
+
+  - description: Tag of the ImageStream to be use
+    displayName: ImageStream Tag
+    required: true
+    name: IMAGE_STREAM_TAG
+    value: "latest"
+
+objects:
+  - kind: CronJob
+    apiVersion: batch/v1beta1
+    metadata:
+      name: graph-backup
+      annotations:
+        thoth-station.ninja/template-version: 0.1.0
+      labels:
+        app: thoth
+        component: graph-backup
+    spec:
+      schedule: "0 * * * *"
+      suspend: ${{THOTH_SUSPEND_JOB}}
+      concurrencyPolicy: Forbid
+      successfulJobsHistoryLimit: 2
+      failedJobsHistoryLimit: 4
+      startingDeadlineSeconds: 5000000
+      jobTemplate:
+        spec:
+          template:
+            metadata:
+              labels:
+                app: thoth
+                component: graph-backup
+            spec:
+              serviceAccountName: "graph-backup-job"
```

Comment:
Is this name okay? or should I modify it as customizable?

Reply:
I think it's OK. :+1: